contracts/adapters/AaveMigrationExtension.sol

```diff
@@ -0,0 +1,615 @@
+        issuanceModule = _issuanceModule;
+        nonfungiblePositionManager = _nonfungiblePositionManager;
+    }
+
+    /* ========== External Functions ========== */
+
+    /**
+     * @notice OPERATOR ONLY: Initializes the Set Token on the Trade Module.
+     */
+    function initialize() external onlyOperator {
+        bytes memory data = abi.encodeWithSelector(tradeModule.initialize.selector, setToken);
+        invokeManager(address(tradeModule), data);
+    }
+    
+    /**
+     * @notice OPERATOR ONLY: Executes a trade on a supported DEX.
+     * @dev Although the SetToken units are passed in for the send and receive quantities, the total quantity
+     * sent and received is the quantity of SetToken units multiplied by the SetToken totalSupply.
+     * @param _exchangeName The human-readable name of the exchange in the integrations registry.
+     * @param _sendToken The address of the token being sent to the exchange.
+     * @param _sendQuantity The amount of the token (in SetToken units) being sent to the exchange.
+     * @param _receiveToken The address of the token being received from the exchange.
+     * @param _minReceiveQuantity The minimum amount of the receive token (in SetToken units) expected from the exchange.
+     * @param _data Arbitrary data used to construct the trade call data.
+     */
+    function trade(
+        string memory _exchangeName,
+        address _sendToken,
+        uint256 _sendQuantity,
+        address _receiveToken,
+        uint256 _minReceiveQuantity,
+        bytes memory _data
+    )
+        external
+        onlyOperator
+    {
+        _trade(
+            _exchangeName,
+            _sendToken,
+            _sendQuantity,
+            _receiveToken,
+            _minReceiveQuantity,
+            _data
+        );
+    }
+
+    /**
+     * @notice OPERATOR ONLY: Mints a new liquidity position in the Uniswap V3 pool.
+     * @param _amount0Desired The desired amount of token0 to be added as liquidity.
+     * @param _amount1Desired The desired amount of token1 to be added as liquidity.
+     * @param _amount0Min The minimum amount of token0 to be added as liquidity.
+     * @param _amount1Min The minimum amount of token1 to be added as liquidity.
+     * @param _tickLower The lower end of the desired tick range for the position.
+     * @param _tickUpper The upper end of the desired tick range for the position.
+     * @param _fee The fee tier of the Uniswap V3 pool in which to add liquidity.
+     * @param _isUnderlyingToken0 True if the underlying token is token0, false if it is token1.
+     */
+    function mintLiquidityPosition(
+        uint256 _amount0Desired,
+        uint256 _amount1Desired,
+        uint256 _amount0Min,
+        uint256 _amount1Min,
+        int24 _tickLower,
+        int24 _tickUpper,
+        uint24 _fee,
+        bool _isUnderlyingToken0
+    )
+        external
+        onlyOperator
+    {
+        // Sort tokens and amounts
```

Comment:
Nit: Even though we are not reusing this in internal calls, might still be worth wrapping the logic in an internal `_mintLiquidityPosition` method for visual consistency when reading through the external methods. (Gas costs of internal function jump is negligble I think. 
Totally optional though.

Reply:
added here [7359b35](https://github.com/IndexCoop/index-coop-smart-contracts/pull/162/commits/7359b35d8d04a97fb7d2d8cb473df04584713b28)